build/Utils.cmake

```diff
@@ -151,3 +152,48 @@ function(extract_sources sources_file)
     endif()
   endif()
 endfunction()
+
+# Determine the path to buck2, taking in order of preference: explicitly
```

Comment:
Please describe the concrete action performed by this function; i.e., it appears to set the BUCK2 cmake variable to the buck2 path that it chooses.